app/controllers/catalog_controller.rb

```diff
@@ -49,5 +49,11 @@ class CatalogController < ApplicationController
     config.add_results_collection_tool(:view_type_group)
 
     config.view.masonry.title_only_by_default = true
+
+    config.add_index_field(
+      'spotlight_upload_description_tesim',
```

Comment:
Minor, but maybe we could swap this string for `Spotlight::Engine.config.upload_description_field` to allow that field name to change if we ever decide?

Reply:
great idea!